src/mailadm/bot.py

```diff
@@ -36,16 +36,42 @@ def __init__(self, db: DB, account: deltachat.Account):
         self.account = account
         with self.db.read_connection() as conn:
             config = conn.config
-            self.admingrpid = config.admingrpid
-            self.admingroup = account.get_chat_by_id(int(self.admingrpid))
+            self.admingrpid = int(config.admingrpid)
+            self.admingroup = account.get_chat_by_id(self.admingrpid)
 
     @account_hookimpl
     def ac_incoming_message(self, message: deltachat.Message):
         arguments = message.text.split(" ")
-        print("process_incoming command:", arguments)
+        print("process_incoming message:", message.text)
         if not self.check_privileges(message):
-            message.create_chat()
-            message.chat.send_text("Sorry, I only take commands from the admin group.")
+            chat = message.create_chat()
+            if chat.is_group():
+                if message.get_sender_contact() not in self.admingroup.get_contacts():
+                    chat.send_text("Sorry, I'm a strictly non-group bot. You can talk to me 1:1.")
```

Comment:
took me a while to figure out. 
```suggestion
                    chat.send_text("Sorry, you can not contact me in a group chat. Please use a 1:1 chat.")
```

Reply:
because the admin bot is also handling groups, i was at first confused when it said "i am strictly non-group bot" :)